### DISCUSSION TITLE
Povratna informacija na drugu provjeru projekta

### DISCUSSION THREAD

**Discussion by dperas** [AIR-FOI-HR/AIR2206#2] [General]
Poštovani,
u nastavku su povratne informacije na vaš rad na projektu.
Greške iz prve provjere još nisu ispravljene (arhitektura sustava nije izrađena do razine modula, nedostaje dizajn ekrana i wireframe, korisničke priče su dosta šture). Issues i epics u Jiri nisu ispravno podešeni, te nisu definirani sprintevi. Također, ne postoji roadmap, odnosno nije definiran. Planirano je premalo sprinteva (samo 2, optimalno bi bilo 4-5), te nijedan nije zadovoljavajuć (burndown chart). Na projektu nije rađeno kontinuirano (ni dokumentacija ni kod nisu pisani redovito). Ne postoji kod na Git-u.
Dakle, broj bodova koji ste ostvarili u ovoj fazi je 3 (od max 20), jer prema onome što ste objavili, niste napravili gotovo ništa. Molim Vas da javite ukoliko imate neki problem ili ste negdje zapeli.

**Comment by denis137255** [comment]
Poštovana profesorice,

Isprike na ovako kasnom odgovoru, nismo očekivali povratnu informaciju. Naime, prošli tjedan sam bio na konzultacijama kod profesora Stapića gdje smo zamolili da se obrana (kao i generalna provedba projekta) pomjeri do 6. mjeseca. Ako se dobro sjećam tada ste bili na bolovanju pa je moguće da se ta informacija nije prenijela. 

Nadam se da nije nastala previše neugodna situacija od ovog te da ste u prilici prokomentirati ovo s profesorom kako bi potvrdili ranije navedeno.

Srdačan pozdrav!

Tim 2206
